src/js/view/controls/components/timeslider.ts

```diff
@@ -25,16 +25,20 @@ const ARIA_TEXT_UPDATE_INTERVAL_MS = 1000;
 const ARIA_TEXT_UPDATE_TIMES = 4;
 
 class TimeTipIcon extends TooltipIcon {
-    text?: HTMLElement;
+    text1?: HTMLElement;
+    text2?: HTMLElement;
     img?: HTMLElement;
     containerWidth?: number;
     container?: HTMLElement;
     textLength?: number;
     dragJustReleased?: boolean;
 
     setup(): void {
-        this.text = document.createElement('span');
-        this.text.className = 'jw-text jw-reset';
+        this.text1 = document.createElement('span');
+        this.text1.className = 'jw-text jw-reset';
+        this.text2 = document.createElement('span');
+        this.text2.className = 'jw-text jw-reset';
+        this.text2.style.display = 'none';
```

Comment:
Wondering if we can use styling like `whitespace: pre or pre-line` or innerHTML instead of textContent so we can keep one text variable?

Reply:
After talking with @zetagame, I added the classes in the latest commit.